README.md

```diff
@@ -191,15 +191,16 @@ defaults defined in the project itself.  See [`reference.conf`](./src/main/resou
-| `lookup-table-size`    | `60`               | The maximum window size of the look up table **per partition**                                                                        |
-| `client-group-id`      | `kafkalagexporter` | Consumer group id of kafka-lag-exporter's client connections                                                                          |
-| `kafka-client-timeout` | `10 seconds`       | Connection timeout when making API calls to Kafka                                                                                     |
-| `clusters`             | `[]`               | A statically defined list of Kafka connection details.  This list is optional if you choose to use the Strimzi auto-discovery feature |
-| `watchers`             | `{}`               | Settings for Kafka cluster "watchers" used for auto-discovery.                                                                        |
+| Key                           | Default            | Description                                                                                                                           |
+|-------------------------------|--------------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| `port`                        | `8000`             | The port to run the Prometheus endpoint on                                                                                            |
+| `poll-interval`               | `30 seconds`       | How often to poll Kafka for latest and group offsets                                                                                  |
+| `lookup-table-size`           | `60`               | The maximum window size of the look up table **per partition**                                                                        |
+| `client-group-id`             | `kafkalagexporter` | Consumer group id of kafka-lag-exporter's client connections                                                                          |
+| `kafka-client-timeout`        | `10 seconds`       | Connection timeout when making API calls to Kafka                                                                                     |
+| `clusters`                    | `[]`               | A statically defined list of Kafka connection details.  This list is optional if you choose to use the Strimzi auto-discovery feature |
+| `watchers`                    | `{}`               | Settings for Kafka cluster "watchers" used for auto-discovery.                                                                        |
+| `prometheus-metric-whitelist` | `.*`               | Regex of metrics to be exposed via Prometheus endpoint. Eg. `^kafka_consumergroup_group_max_lag.*`                                      |
```

Comment:
It's probably redundant to call this the `prometheus-metric-whitelist`.  Can you rename it to just `metric-whitelist`, along with the other variations of the name used in this PR?

Reply:
It may be easier for end users to configure this if it were an array of regular expressions that can match the white list.  What do you think?

Reply:
Yes, array of regular expressions will make this consistent with the approach for whitelisted topic as well. Will implement that.

As for the CLA process, i have tried several times but still not successful. I suspect this might be due to me using a different user for the commit than the user i used to submit the PR. May I close this PR and raise another one (after addressing the comments), to see if the CLA process passed?

Reply:
@shuheng-gojek Sure, go ahead and recreate the PR as your convenience.